lib/Log/Dispatchouli/Proxy.pm

```diff
@@ -179,9 +165,7 @@ sub fmt_event {
   return join q{ }, @$kv_aref;
 }
 
-sub log_event {
-  my ($self, $event, $data) = @_;
-
+sub log_event ($self, $event, $data) {
```

Comment:
log_event can be used with an event but no data. This will break consumers of this I believe